instrumentation-api-incubator/src/main/java/io/opentelemetry/instrumentation/api/incubator/semconv/http/HttpExperimentalMetricsAdvice.java

```diff
@@ -32,7 +32,8 @@ static void applyClientRequestSizeAdvice(LongHistogramBuilder builder) {
                 NetworkAttributes.NETWORK_PROTOCOL_NAME,
                 NetworkAttributes.NETWORK_PROTOCOL_VERSION,
                 ServerAttributes.SERVER_ADDRESS,
-                ServerAttributes.SERVER_PORT));
+                ServerAttributes.SERVER_PORT,
+                UrlAttributes.URL_SCHEME));
```

Comment:
this is opt-in, so users would need to provide their own metric view to override this